sockeye/constants.py

```diff
@@ -88,6 +88,10 @@
 
 MEASURE_SPEED_EVERY = 50  # measure speed and metrics every X batches
 
+# Monitor constants
+STAT_FUNC_NORM = "norm"  # default MXNet monitor stat func: |x|/size(x)
+MONITOR_STAT_FUNCS = {STAT_FUNC_NORM: None}
```

Comment:
I guess it's for implicitly using the mxnet default defined inside of Monitor. What if mxnet decides to change the norm? Would it maybe make more sense to all the option "default" or "mx_default"?
```
STAT_FUNC_DEFAULT = "mx_default"  # default MXNet monitor stat func: |x|/size(x)
MONITOR_STAT_FUNCS = {STAT_FUNC_DEFAULT: None, STAT_FUNC_NORM: some_other_func}
```